web/README.md

```diff
@@ -1 +1,4 @@
 # trading-card-manager
+
+# Testing
```

Comment:
To com preguiça de explicar bem agora, deixando somente um lembrete pro meu futuro eu